### PR TITLE
OCT-1971: Cleaning up client after upgrading to v1.5

### PR DESCRIPTION
**Pull request by jmikolajczyk**
## Definition of Done

1. [ ] If required, the desciption of your change is added to the [QA changelog](https://www.notion.so/octantapp/Changelog-for-the-QA-d96fa3b411cf488bb1d8d9a598d88281) 
2. [ ] Acceptance criteria are met.
3. [ ] PR is manually tested before the merge by developer(s).
    - [ ] Happy path is manually checked.
4. [ ] PR is manually tested by QA when their assistance is required (1).
    - [ ] Octant Areas & Test Cases are checked for impact and updated if required (2).
5. [ ] Unit tests are added unless there is a reason to omit them.
6. [ ] Automated tests are added when required.
7. [ ] The code is merged.
8. [ ] Tech documentation is added / updated, reviewed and approved (including mandatory approval by a code owner, should such exist for changed files).
    - [ ] BE: Swagger documentation is updated.
9. [ ] When required by QA:
    - [ ] Deployed to the relevant environment.
    - [ ] Passed system tests.

---

(1) Developer(s) in coordination with QA decide whether it's required. For small tickets introducing small changes QA assistance is most probably not required.

(2) [Octant Areas & Test Cases](https://docs.google.com/spreadsheets/d/1cRe6dxuKJV3a4ZskAwWEPvrFkQm6rEfyUCYwLTYw_Cc).

### CODE REVIEW

**Review by aziolek — APPROVED** [review]

File: client/src/components/Home/HomeGridCurrentGlmLock/ModalLockGlm/LockGlmNotification/LockGlmNotification.tsx [hunk 1]
```diff
@@ -53,11 +53,7 @@ const LockGlmNotification: FC<LockGlmNotificationProps> = ({
       isVertical
     >
       <div className={styles.notification}>
-        <Svg
-          classNameWrapper={styles.iconWrapper}
```
Comment (aziolek) [inline client/src/components/Home/HomeGridCurrentGlmLock/ModalLockGlm/LockGlmNotification/LockGlmNotification.tsx:57]: Usage of class is removed, but not the class?

File: client/src/components/Home/HomeGridTransactions/HomeGridTransactions.tsx [hunk 2]
```diff
@@ -43,7 +43,7 @@ const HomeGridTransactions: FC<HomeGridTransactionsProps> = ({ className }) => {
     <GridTile className={className} title={t('transactions')}>
       <div className={styles.root}>
         {showLoader ? (
-          <div className={styles.skeleton}>
+          <div>
             <TransactionsSkeleton />
           </div>
```
Comment (aziolek) [inline client/src/components/Home/HomeGridTransactions/HomeGridTransactions.tsx:48]: Do we even need to wrap it now?

File: client/src/components/Metrics/MetricsEpoch/MetricsEpoch.tsx [hunk 3]
```diff
@@ -156,13 +132,6 @@ const MetricsEpoch = (): ReactElement => {
           totalPersonal={totalPersonal}
           totalUserDonations={sumOfDonations}
         />
-        {epoch < 4 && (
-          <MetricsEpochGridBelowThreshold
-            className={styles.belowThreshold}
-            ethBelowThreshold={ethBelowThreshold}
-            isLoading={isLoading}
-          />
-        )}
```
Comment (aziolek) [inline client/src/components/Metrics/MetricsEpoch/MetricsEpoch.tsx:165]: We won't show historic ETH below threshold at all?

**Review by aziolek — CHANGES_REQUESTED** [review]

File: client/src/components/Metrics/MetricsEpoch/MetricsEpoch.tsx (hunk 3 above)
Comment (jmikolajczyk) [reply client/src/components/Metrics/MetricsEpoch/MetricsEpoch.tsx:165]: There is no place in new design for that (new row with one tile).

File: client/src/components/Home/HomeGridCurrentGlmLock/ModalLockGlm/LockGlmNotification/LockGlmNotification.tsx (hunk 1 above)
Comment (jmikolajczyk) [reply client/src/components/Home/HomeGridCurrentGlmLock/ModalLockGlm/LockGlmNotification/LockGlmNotification.tsx:57]: It was use of non-existent class.

File: client/src/components/Home/HomeGridTransactions/HomeGridTransactions.tsx (hunk 2 above)
Comment (jmikolajczyk) [reply client/src/components/Home/HomeGridTransactions/HomeGridTransactions.tsx:48]: We don't

**Review by aziolek — APPROVED** [review]

**Pull request merged**